### DISCUSSION TITLE
plot of optics evolution along a single element

### DISCUSSION THREAD

**Discussion by simoneliuzzo** [atcollab/at#848] [General]
Dear @lfarv  and @swhite2401 ,

when asking to `atplot` to produce optics in transferline mode, the case of a lattice composed of a single element seems not an option for `atplot`. Below an example:

```

optics_struct.beta = [2,1];
optics_struct.alpha = [0,0];
optics_struct.mu = [0,0];
optics_struct.ClosedOrbit = [0,0,0,0];
optics_struct.Dispersion = [0,0,0,0]';

% FAILS
%figure; atplot({atdrift('D',1)}, 'twiss_in', optics_struct);

% WORKS
figure; atplot({atdrift('D',1);atdrift('D',1)}, 'twiss_in', optics_struct);


```

Since I noticed this only now, I posted this message. May be it is expected. 

thank you
best regards
Simone

**Comment by lfarv** [comment]
This is fixed in #854.
